src/libs/ReportUtils.js

```diff
@@ -4418,7 +4418,7 @@ export {
     getTransactionReportName,
     getTransactionDetails,
     getTaskAssigneeChatOnyxData,
-    getParticipantsIDs,
```

Comment:
how do we know no other fork/branch has added calls to this method that might end up breaking once we merge this?

Reply:
I think a safer way to approach this would be to leave `getParticipantsIDs` in the code and just add `getVisibleMemberIDs` as a new method even if it' basically just duplicating the logic.

Add a `deprecated` comment to `getParticipantsIDs` and a console log and/or log to server or throw that says "this is deprecated use `getVisibleMemberIDs` instead. 

Then after we merge this and are sure there are no other usages of this method then fully remove it.

Reply:
Since I am renaming the `visibleChatMemberList` and that requires Auth and Web-E PRs, I think we have time to make this change and it seems worth it, I will add it to the PR.

Reply:
If someone has a fork of this repo they should hopefully get conflicts and/or see their code break 🤞